src/components/TagContainer.js

```diff
@@ -0,0 +1,25 @@
+import TagIcon from "./TagIcon";
+
+function TagContainer({ tag, onClick = undefined, feed = false, unremovable = false }) {
+  return (
+    <div>
+    {
+      !feed ?
+      <span className="flex m-2 text-white justify-between items-center text-xs sm:text-sm bg-gray-400 rounded-full px-3 py-1 font-bold leading-loose">
```

Comment:
I think this code can be simplified by doing inline conditionals for the classes. 
https://stackoverflow.com/questions/30533171/react-js-conditionally-applying-class-attributes

Reply:
Done, I used conditionals inline style.
``` JavaScript
<span className={ feed ? feedTagStyle : createTagStyle } />
```